ZenPacks/zenoss/HttpMonitor/datasources/HttpMonitorDataSource.py

```diff
@@ -1,157 +1,208 @@
 ##############################################################################
-# 
-# Copyright (C) Zenoss, Inc. 2008, all rights reserved.
-# 
+#
+# Copyright (C) Zenoss, Inc. 2018, all rights reserved.
+#
 # This content is made available according to terms specified in
 # License.zenoss under the directory where your Zenoss product is installed.
-# 
+#
 ##############################################################################
 
 
-__doc__='''HttpMonitorDataSource.py
+__doc__ = '''HttpMonitorDataSource.py
 
 Defines datasource for HttpMonitor
 '''
 
-import Products.ZenModel.RRDDataSource as RRDDataSource
-from Products.ZenModel.ZenPackPersistence import ZenPackPersistence
-from AccessControl import ClassSecurityInfo, Permissions
-from Products.ZenUtils.Utils import binPath
+import time
+from twisted.internet import defer
+from zope.interface import implementer
 
+import httplib
```

Comment:
Since you're organizing the imports, `import httplib` should be placed on the line before `import time`.  `httplib` is a standard Python module, so it makes sense to group it with the other standard Python module `import` statements.